01 - Community/People/getmatterapp.md

```diff
@@ -0,0 +1,57 @@
+---
+aliases:
+- Matter
```

Comment:
I'm not sure if this matters (no pun intended) but we now have:

- A person whose file is called getmatterapp, but whose alias is Matter
- A plugin whose file is called matter, and whose alias is also Matter

Because the file names are different, I think it's OK - but wanted to point it out just in case.

Reply:
I think with aliases, we might be safe (unlike the hipstersmoothie case)